2023.csv

```diff
@@ -17,7 +17,7 @@ PyCon Taiwan,2023-09-02,2023-09-03,"Taipei, Taiwan",TWN,,,,https://tw.pycon.org/
 Kiwi PyCon XII,2023-09-15,2023-09-17,"Waihōpai (Invercargill), Aotearoa, New Zealand",NZL,Ascot Park Hotel,,2023-05-19,https://kiwipycon.nz/,https://kiwipycon.nz/call-for-proposals,https://kiwipycon.nz/sponsorship
 PyCon Uganda,2023-09-21,2023-09-23,"MoTIV, Kampala, Uganda",UGA,,,,https://ug.pycon.org,https://ug.pycon.org/speakers,https://ug.pycon.org/sponsors
 PyCon UK,2023-09-22,2023-09-25,"Cardiff, United Kingdom",GBR,Cardiff City Hall,,,https://2023.pyconuk.org/,,https://2023.pyconuk.org/sponsorship/
-PyGotham TV 2023,2023-10-06,2023-10-07,"New York, New York, United States of America",USA,Online,,2023-05-15,https://2023.pygotham.tv,https://cfp.pygotham.tv,
```

Comment:
Was this accidental?

Reply:
Sure was! (I'm an organizer for PyGotham and was using that row as a test, oops.)

Reply:
Thanks for catching that. Fixed.